src/main/java/org/opentripplanner/model/impl/OtpTransitServiceBuilder.java

```diff
@@ -103,6 +106,14 @@ public OtpTransitServiceBuilder() {
         return frequencies;
     }
 
+    /**
+     * get multimap of Notices by the TransitEntity id (Multiple types; hence the Serializable). Entities
+     * that might have Notices are Routes, Trips, Stops and StopTimes.
+     */
+    public Multimap<Serializable, Notice> getNoticeAssignments() {
```

Comment:
In Netex, are IDs per-type unique or feed-level unique? If they are not feed unique, you can get a collision between a route and a stop or trip. This is one reason to key on the objects themselves instead of their IDs, and maybe use identity instead of semantic or ID-based equality and hashcodes.